app-text/aobook/aobook-1.0.3.recipe

```diff
@@ -4,41 +4,43 @@ of Japanese-language literature."
 HOMEPAGE="https://osdn.net/projects/aobook/"
 COPYRIGHT="2014-2017 Azel"
 LICENSE="BSD (3-clause)"
-REVISION="2"
-localRevision="3"
+REVISION="3"
+localRevision="4"
 SOURCE_URI="https://github.com/hanya/aobook-haiku/archive/v$portVersion-$localRevision.tar.gz"
```

Comment:
I've been thinking about something we could do: rename the recipe to `aobook-1.0.3_4.recipe`.
It would allow to "insert" the `localRevision` inside the `portVersion`. This would require the following cosmetic changes:
```diff
-REVISION="3"
+REVISION="1"
-localRevision="4"
-SOURCE_URI="https://github.com/hanya/aobook-haiku/archive/v$portVersion-$localRevision.tar.gz"
+REVISION="1"
+SOURCE_URI="https://github.com/hanya/aobook-haiku/archive/v${portVersion/_/-}.tar.gz"
 CHECKSUM_SHA256="f840a98b0f0c1f3fa31ae19498bc9051106e068b9af95eb4065643693f7f1e07"
+SOURCE_FILENAME="aobook-haiku-${portVersion/_/-}.tar.gz"
-SOURCE_DIR="aobook-haiku-$portVersion-$localRevision"
+SOURCE_DIR="aobook-haiku-${portVersion/_/-}"
```

I think this could be a good idea. What do you think?

(Well, the `SOURCE_FILENAME` is not a requirement for changing the name of the recipe but a general recommendation.)

In case you prefer to keep the internalRevision, that's also OK.
(In that case, could you please insert a `SOURCE_FILENAME="aobook-haiku-$portVersion-$localRevision.tar.gz"` between `CHECKSUM_SHA256` and `SOURCE_DIR` ?)